### PR TITLE
Bring Comparison View up to spec

### PR DESCRIPTION
**Pull request by chutten**
This PR factors out our bar plot customizations and specializations into their own component (creatively named `BarPlot`) and then converts both Distribution and Comparison views to use it for a consistent plot experience.

Oh, and it enables comparing all the types of probes now, too. (Architecture!)

As before, live preview's here once [ci completes](https://travis-ci.org/chutten/mdv2/builds/428225676): https://chutten.github.io/mdv2/

### CODE REVIEW

File: src/components/barplot.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      xaxis: {
+        type: "category",
+      },
+      yaxis: {
+        title: "Proportion of Users",
+        hoverformat: ".4p",
+        tickformat: ".3p",
+      },
+    };
+    return (
```
Comment (georgf) [inline src/components/barplot.jsx]: Style nit: empty line before return for visual separation?

File: src/components/distributionview.jsx [hunk 2]
```diff
@@ -116,22 +42,8 @@ export class DistributionView extends Component {
...
-                  title: metric,
-                },
-                yaxis: {
-                  title: "Proportion of Users",
-                  hoverformat: ".4p",
-                  tickformat: ".3p",
-                },
-              } }
+            <BarPlot
+              data={[this.props.dataStore.active]}
```
Comment (georgf) [inline src/components/distributionview.jsx:46]: As a possible P2/P3 follow-up:
Can we improve that components interface a bit for readability?
E.g. giving it different props like `metricName`, `metricData`, `compareMetricData`?

**Review by georgf — APPROVED** [review]
Looking great, this is really starting to come together!
Also, components, they work out well!

File: src/components/distributionview.jsx (hunk 2 above)
Comment (chutten) [reply src/components/distributionview.jsx:46]: #75 filed

**Pull request merged**